config/base.yaml

```diff
@@ -12,6 +12,7 @@ security-server:
     owner_member_class: <MEMBER_CLASS>
     owner_member_code: <MEMBER_CODE>
     security_server_code: <SERVER_CODE>
+    software_token_id: 0 # default software token ID is zero
```

Comment:
To conform with the proposed and discussed configuration style, please substitute with <SOFTWARE_TOKEN_ID>

Reply:
AFAIU it is never different from zero for default software token, so rather would keep the default value, changing it as follows:

```
software_token_id: 0 # <SOFT_TOKEN_ID> -- default software token ID is zero
```
I also thought it might be better to rename it to ``default_software_token_id``, but unprefixed ``software_token_pin`` was already in place, so I decided to withhold that until we have multi-token operations in place, to see how the naming will fit then.